pkg/ttnpb/field_mask_validation.go

```diff
@@ -79,6 +79,11 @@ var AllowedFieldMaskPathsForRPC = map[string][]string{
 	"/ttn.lorawan.v3.ApplicationPubSubRegistry/List": ApplicationPubSubFieldPathsNested,
 	"/ttn.lorawan.v3.ApplicationPubSubRegistry/Set":  ApplicationPubSubFieldPathsNested,
 
+	// Application Packages:
+	"/ttn.lorawan.v3.ApplicationPackageRegistry/Get":  ApplicationPackageAssociationFieldPathsNested,
+	"/ttn.lorawan.v3.ApplicationPackageRegistry/List": ApplicationPackageAssociationFieldPathsNested,
+	"/ttn.lorawan.v3.ApplicationPackageRegistry/Set":  ApplicationPackageAssociationFieldPathsNested,
```

Comment:
Is this complete?

Reply:
`DeleteAssociation` and `List` do not contain field masks, what is missing ?